coops-ui/src/main.ts

```diff
@@ -28,6 +28,24 @@ Vue.use(Affix)
 
 const vuetify = new Vuetify({ iconfont: 'mdi' })
 
+function resetAuth () {
```

Comment:
What's this? It's not documented as a change in this PR's description or the ticket description.

Reply:
Yikes, I left some debug code in...good catch, Thanks!